vms/components/gas/state.go

```diff
@@ -18,27 +18,29 @@ type State struct {
 	Excess   Gas `serialize:"true" json:"excess"`
 }
 
-// AdvanceTime adds maxPerSecond to capacity and subtracts targetPerSecond
+// AdvanceTime adds capacityRate to capacity and subtracts targetRate
 // from excess over the provided duration.
+// The units chosen for time must be consistent with the units chosen for capacityRate and targetRate.
```

Comment:
This file wraps text at 80 characters
```suggestion
// AdvanceTime adds capacityRate to capacity and subtracts targetRate from
// excess over the provided duration.
//
// The units chosen for time must be consistent with the units chosen for
// capacityRate and targetRate.
```